experiment_prototype/experiment_prototype.py

```diff
@@ -1616,13 +1642,15 @@ def check_slice(self, exp_slice):
 
                 # Check if any scanbound times are shorter than the intt.
                 if len(exp_slice['scanbound']) == 1:
-                    if exp_slice['scanbound'][0] * 1000 < exp_slice['intt']:
+                    tolerance = 1e-9
+                    if exp_slice['intt'] > (exp_slice['scanbound'][0] * 1000 + tolerance):
                         error_list.append("Slice {} intt longer than "
                             "scanbound times".format(exp_slice['slice_id']))
                 else:
                     for i in range(len(exp_slice['scanbound']) - 1):
-                        if ((exp_slice['scanbound'][i+1] - exp_slice['scanbound'][i]) * 1000 <
-                            exp_slice['intt']):
+                        tolerance = 1e-9
+                        beam_time = (exp_slice['scanbound'][i+1] - exp_slice['scanbound'][i]) * 1000
+                        if exp_slice['intt'] < beam_time + tolerance:
```

Comment:
I think this is the wrong sign 

```suggestion
                        if exp_slice['intt'] > beam_time + tolerance:
```